chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/LeftRoomTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.LeftRoom
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class LeftRoomTest : Spek({
+
+    describe("given a joined rooms state of two rooms") {
+        val givenState = ChatkitState(
+                joinedRoomsState = JoinedRoomsState(
+                        mapOf(
+                                JoinedRoomsStateTestUtil.roomOneId to JoinedRoomsStateTestUtil.roomOne,
+                                JoinedRoomsStateTestUtil.roomTwoId to JoinedRoomsStateTestUtil.roomTwo
+                        ),
+                        mapOf(
```

Comment:
named param for unread count would help with readability

Reply:
👍 have added more named parameters where i thought it might not be very clear.